zospy/analyses/mtf.py

```diff
@@ -237,3 +237,123 @@ def fft_through_focus_mtf_fromcfg(
     )
 
     return analysis.complete(oncomplete, result)
+
+
+def huygens_mtf(
+    oss: OpticStudioSystem,
+    pupil_sampling: str | int = "32x32",
+    image_sampling: str | int = "32x32",
+    image_delta: float = 0.0,
+    wavelength: str | int = "All",
+    field: str | int = "All",
+    mtftype: constants.Analysis.Settings.Mtf.HuygensMtfTypes | str = "Modulation",
+    maximum_frequency: float = 150.0,
+    use_polarization: bool = False,
+    use_dashes: bool = False,
+    oncomplete: OnComplete | str = OnComplete.Close,
+) -> AnalysisResult:
+    """Wrapper around the OpticStudio Huygens MTF.
+
+    For an in depth explanation of the parameters, see the Zemax OpticStudio user manual
+
+    Parameters
+    ----------
+    oss : zospy.core.OpticStudioSystem
+        A ZOSPy OpticStudioSystem instance. Should be sequential.
+    pupil_sampling : str | int
+        The pupil sampling, either string (e.g. '64x64') or int.
+        The integer will be treated as a ZOSAPI Constants integer.
+    image_sampling : str | int
+        The image sampling, either string (e.g., '64x64') or int.
+        The integer will be treated as a ZOSAPI Constants integer.
+    image_delta : float
+        The Image Delta, defaults to 0.0.
+    wavelength : str | int
+        The wavelength to use in the MTF. Either 'All' or an integer specifying the wavelength number.
+    field : str | int
+        The field to use in the MTF. Either 'All' or an integer specifying the field number.
+    mtftype : zospy.constants.Analysis.Settings.Mtf.MtfTypes.Modulation
+        The MTF type (e.g. `Modulation`) that is calculated.
+    maximum_frequency : float
+        The maximum frequency at which the MTF is calculated.
+        Units are either cycles/mm or cycles/mr, depending on system setting).
```

Comment:
Unmatched closing parenthesis:

> ```setting).```

Reply:
Fixed in https://github.com/MREYE-LUMC/ZOSPy/pull/55/commits/d566fe3322ca9ca320e498f5c30960894103b553.